File: intelligence.java
import java.util.Scanner;
public class Main {
	static void intelligence(boolean is_a_failure, String name) {
		if (is_a_failure == true) {
			int randomnum2 = (int)(Math.random()* 5);
			if (randomnum2 == 0) {
				System.out.println(name + " is stupid");
			} else if (randomnum2 == 1) {
				System.out.println(name + " is maidenless");
			} else if (randomnum2 == 2) {
				System.out.println(name + " needs to touch grass");
			} else if (randomnum2 == 3) {
				System.out.println("your mom");
			} else if (randomnum2 == 4) {
				System.out.println(name + " is short");
			}
		} else {
				System.out.println(name + " is smart");
		}
	}
	public static void main(String[] args) {
		Scanner usernameinput = new Scanner(System.in);
		System.out.println("Whats their Name?:");
		String Username = usernameinput.nextLine();
		int randomnum = (int)(Math.random() * 101);
		System.out.println("their score is " + randomnum);
		if (randomnum <= 50) {
			intelligence(true, Username);
		} else if (randomnum > 50) {
			intelligence(false, Username);
		} 	
	}
}

File: intelligence.py
from random import random
import string
name = input("whats your name?:")
randomnum2 = int(random()* 5)
randomnum = int(random()* 101)
smart = "null"
if randomnum >= 50:
    smart = True
else:
    smart = False
if smart == True:
    print("their score is " + str(randomnum))
    print(name + " is smart")
if smart == False:
    if randomnum2 == 0:
        print("their score is " + str(randomnum))
        print(name + " is stupid")
    if randomnum2 == 1:
        print("their score is " + str(randomnum))
        print(name + " is maidenless")
    if randomnum2 == 2:
        print("their score is " + str(randomnum))
        print(name + " needs to touch grass")
    if randomnum2 ==3:
        print("their score is " + str(randomnum))
        print("your mom")
    if randomnum2 ==4:
        print("their score is " + str(randomnum))
        print(name + " is short")
